src/components/AppHeader/NavigationBar/NavigationBar.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+import Button from "../../common/Button";
+
+import VARIANTS from "../../../constants/variants";
+import LANGUAGES from "../../../constants/languages";
+
+const { TOOL } = VARIANTS;
+
+const {
+  ALL,
+  PYTHON,
+  JAVA,
+  JAVASCRIPT,
+  CSHARP,
+  C_CPP,
+  PHP,
+  R,
+  OBJECTIVE_C,
+  OTHERS
+} = LANGUAGES;
+
+const NavigationWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 0 20px;
+`;
+
+function NavigationBar() {
+
+  const languageList = [ALL, PYTHON, JAVA, JAVASCRIPT, CSHARP, C_CPP, PHP, R, OBJECTIVE_C, OTHERS];
+
+  return (
+    <NavigationWrapper>
+      {languageList.map((language) => {
+        return <Button key={language} variant={TOOL} >{language}</Button>;
+      })}
```

Comment:
이전에 회의한 바와 같이 스타일을 통일하면 좋을 것 같습니다!

```suggestion
      {languageList.map((language) => (
        <Button key={language} variant={TOOL}>{language}</Button>
      ))}
```